electrum/gui/qt/channels_list.py

```diff
@@ -84,10 +84,14 @@ def task():
         WaitingDialog(self, 'please wait..', task, self.on_success, self.on_failure)
 
     def force_close(self, channel_id):
-        def task():
-            coro = self.lnworker.force_close_channel(channel_id)
-            return self.network.run_from_another_thread(coro)
-        if self.parent.question('Force-close channel?\nReclaimed funds will not be immediately available.'):
+        if self.lnworker.wallet.is_lightning_backup():
+            msg = _('<b>WARNING<b>: force-closing from an old state might result in fund loss.\nAre you sure?')
```

Comment:
</b> to close the tag
Also, `self.parent.question` needs to be passed `rich_text=True` I guess?
and also `\n` might not work in rich text; not sure.